src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj

```diff
@@ -2,21 +2,31 @@
 
   <PropertyGroup>
     <Configurations>Debug;Release;Debug-Intrinsics;Release-Intrinsics</Configurations>
-    <UseIntrinsics Condition="'$(UseIntrinsics)' == ''">$(Configuration.EndsWith('-Intrinsics'))</UseIntrinsics>
-    
     <TargetFramework Condition="'$(UseIntrinsics)' != 'true'">netstandard2.0</TargetFramework>
     <TargetFrameworks Condition="'$(UseIntrinsics)' == 'true'">netstandard2.0;netcoreapp3.0</TargetFrameworks>
     <IncludeInPackage>Microsoft.ML.CpuMath</IncludeInPackage>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>$(DefineConstants);CORECLR;PRIVATE_CONTRACTS</DefineConstants>
+    <LangVersion>7.3</LangVersion>
+    <!-- The SDK we currently restore and use has 2.0 as the maximum supported version -->
+    <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
```

Comment:
This can be removed.

Reply:
Agree with removing the `NETCoreAppMaximumVersion` tag.  

In a recent fix of the `VectorSum` function in `SseIntrinsics.cs`, since we need to rely on the read-only feature of `Vector128` introduced in C# 7.2 to change the vector in-place, the Lang Version has to be at least 7.2.  However, since we use the feature of `extensible fixed statements` in `SseIntrinsics.cs`, which is a feature in C# 7.3, the `LangVersion` restriction has been set to `7.3`.  Reference: https://github.com/dotnet/roslyn/pull/24469

Please let me know how I should proceed.

Reply:
sorry, I was specifically talking about `<NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>`.  The `LangVersion` change is needed.